src/App.js

```diff
@@ -239,6 +244,25 @@ class App extends React.Component {
         log('transaction bag', transactionBag)
         this.setState({ transactionBag })
       },
+      onIdentityIntent: async identityIntent => {
+        // callback for both iframe and native apps
+        if (identityIntent.providerName === 'local') {
```

Comment:
I don't think we need this check. `resolveAddressIdentity` will throw an error if a provider doesn't exist